spec/models/announcement_spec.rb

```diff
@@ -4,38 +4,18 @@
   let(:today) { Time.zone.now }
 
   it "it has a tag with a title of 'news' ", js: true do
-    tag1 = Refinery::Tags::Tag.create(title: 'news', narrative: "We do announcements.", tag_type: "content_type")
-    announcement1 = FactoryBot.create(:announcement, title: 'Test Announcement Title', published_date: today)
-    announcement1.tags.push(tag1)
-    announcement1.save
-
-    expect(announcement1.tags[0].title).to eq('news')
-  end
-
-  it "it has a tag with a tag_type of 'content_type'", js: true do
```

Comment:
Clarifying my understanding: is this test now unnecessary because of the changes to tags we discussed a couple of PRs ago?

Reply:
[Many folks recommend not testing ActiveRecord validations, associations, and scopes](https://signalvnoise.com/posts/3159-testing-like-the-tsa) so I think its ok to just nuke this one. If we were to rewrite it, I would do so from the perspective of submitting a form in the admin panel as an integration test, and not testing the validation alone. Though the main reason I got rid of this is the change to the announcements factory now guarantees the presence of the tags we need.